andela/asgard-rc
# #158755842 Improve Media Gallery

#### What does this PR do?
- Improves media gallery for prpducts

#### Description of Task to be completed?
- change product grid layout
- Add hover effect when product is hovered on

#### How should this be manually tested?
- Setup application with instructions in readme.
- click on a particular tag or image on the landing page
-you should see an improved media gallery


#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#158755842](https://www.pivotaltracker.com/story/show/158755842)

#### Screenshots (if appropriate)
![screen shot 2018-07-27 at 8 21 54 pm 2](https://user-images.githubusercontent.com/26988297/43342505-c725e36a-91da-11e8-989d-c89216678f8b.png)
